pennylane/transforms/qcut.py

```diff
@@ -941,6 +940,33 @@ def circuit(x):
         >>> qml.grad(cut_circuit)(x)
         -0.506395895364911
     """
+    if len(tape.measurements) != 1:
```

Comment:
Do we want to exclude tapes without measurements? Would be fairly simple to return fragment tapes. Though use cases for this is questionable

Reply:
I think it makes sense to exclude the zero-measurements case:
- It seems strange to go through the (potentially huge) process of cut placement and fragment execution, to contract a tensor network to nothing
- I'm also not sure if it works in the current pipeline in practice
- This is anyway quite an edge case because QNodes will always have at least one measurement. So only if a user applies the transform to a tape could they see this warning.
- It's assumed in [this](https://github.com/PennyLaneAI/pennylane/pull/2254) PR